build.gradle

```diff
@@ -11,7 +11,7 @@ plugins {
     id 'io.freefair.lombok' version '8.6'
     id 'org.sonarqube' version '4.4.1.3373'
     id 'pmd'
-    id 'com.github.hmcts.rse-cft-lib' version '0.19.1030'
+    id 'com.github.hmcts.rse-cft-lib' version '0.19.1034'
```

Comment:
Keep this as 1030 - newer versions seem to be broken with WSL

Reply:
Fixed